stdlib/seq.mli

```diff
@@ -81,6 +81,14 @@ val iter : ('a -> unit) -> 'a t -> unit
     The traversal happens immediately and will not terminate on infinite
     sequences. *)
 
+val cons : 'a -> 'a t -> 'a t
+(** [cons x xs] is the sequence containing the element [x] followed by
+    the [xs] sequence @since 4.11 *)
+
+val append : 'a t -> 'a t -> 'a t
+(** [append xs ys] is the sequence [xs] followed by the sequence [ys]
+    @since 4.11 *)
+
```

Comment:
Small nitpick: I think the two functions would be better located functions just after `return` rather than between sequence iterators and unfold.